src/Internals/SectionHeader.tsx

```diff
@@ -12,7 +12,7 @@ export const SectionHeader = styled("div")(({ theme }) => ({
 
   // This ensures that the card header text and card controls are placed in opposite corners.
   justifyContent: "space-between",
-  height: theme.space.element * 2,
+  height: "36px",
```

Comment:
```suggestion
  height: 36,
```

You don't need to specify `px`, a number is enough here 😉

Reply:
thanks! didn't notice that